projects/plugins/boost/app/lib/class-viewport.php

```diff
@@ -149,6 +149,15 @@ public static function get_viewport_size() {
 			$data        = self::get_default_viewport_size_for_device( $device_type );
 		}
 
+		/**
+		 * Filter the viewport size. This is used for testing.
+		 *
+		 * @param array `$data`: array containing the width and height of the view port
+		 * @param string `$cookie_name`: Name of the viewport cookie
+		 * @param string `$cookie_value`: value of the viewport cookie
+		 *
+		 * @since   1.0.0
+		 */
 		return apply_filters( 'jetpack_boost_viewport_size', $data, self::VIEWPORT_COOKIE, $cookie_value );
 	}
 
```

Comment:
I couldn't find any usage of the `Viewport` class. I think we can just get rid of this file and the corresponding test.

Reply:
Done